autotest.pm

```diff
@@ -100,7 +100,7 @@ sub _debug_python_version () {
     state $python_loaded = 0;
     return if $python_loaded++;
     my $code = <<~'EOM';
-    use Inline::Python;
+    require Inline::Python;
```

Comment:
I would say this change is not necessary.
This function is only executed for a `.py` test, and the code is in a string eval.